client/devices/client.go

```diff
@@ -0,0 +1,181 @@
+// Copyright 2021 Northern.tech AS
+//
+//    Licensed under the Apache License, Version 2.0 (the "License");
+//    you may not use this file except in compliance with the License.
+//    You may obtain a copy of the License at
+//
+//        http://www.apache.org/licenses/LICENSE-2.0
+//
+//    Unless required by applicable law or agreed to in writing, software
+//    distributed under the License is distributed on an "AS IS" BASIS,
+//    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//    See the License for the specific language governing permissions and
+//    limitations under the License.
+package devices
+
+import (
+	"crypto/tls"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"net/http/httputil"
+	"strings"
+
+	"github.com/pkg/errors"
+
+	"github.com/mendersoftware/mender-cli/log"
+)
+
+type devicesList struct {
+	devices []deviceData
+}
+
+type deviceData struct {
+	ID           string `json:"id"`
+	IdentityData struct {
+		Mac string `json:"mac"`
+		Sku string `json:"sku"`
+		Sn  string `json:"sn"`
```

Comment:
Mac Sku Sn -- can we be more verbose with the naming?

Reply:
What are they? I followed the naming from API docs

Reply:
Found it, I'll use the full name from:

![image](https://user-images.githubusercontent.com/3168644/109155539-f62ab000-776f-11eb-8164-74ab3df9537e.png)